plugins/dashboard/src/components/DashboardComponent/DashboardComponent.tsx

```diff
@@ -15,7 +15,8 @@ const useStyles = makeStyles({
 export const DashboardComponent = () => {
   const classes = useStyles();
   const [table, setTable] = useState('home');
-  const data = useDashboard();
+  let data: any = {};
```

Comment:
Can you change this in the useDashboard hook instead of here?  Make it so that the hook returns the response or an empty object so any future usage doesn't require the pattern inside the component

Reply:
just added that.